token-holders/participate-in-governance.md

```diff
@@ -0,0 +1,17 @@
+# Participate in governance
```

Comment:
Might wanna capitalize the word since it's the title (which is also the same standard followed by other pages)

Reply:
same for other pages in this PR

Reply:
Done